src/test/test-utils.ts

```diff
@@ -292,3 +292,15 @@ export function change(uri: string, text: string) {
 		}]
 	});
 }
+
+export function completions(params: vscode.TextDocumentPositionParams, expected: string[], done: (err?: Error) => void) {
```

Comment:
I'd rather try to switch to async/promise-style as a part of separate tech debt task